build.gradle.kts

```diff
@@ -185,8 +173,6 @@ if (!isRelease || isMainBranch) { // Only publish releases from the main branch
             channel.set(if (isRelease) "Release" else if (isMainBranch) "Snapshot" else "Alpha")
             changelog.set(changelogContent)
             apiKey.set(System.getenv("HANGAR_SECRET"))
-            owner.set("TheMeinerLP")
-            slug.set("BetterGoPaint")
             platforms {
```

Comment:
Thats going to fail later in the pipeline because Hangar wants to have an ID in the newer version

Reply:
registrations are unique and act as the previously called "slug"
also owner and slug do not exist anymore